src/vcu/src/controls.c

```diff
@@ -0,0 +1,191 @@
+#include "controls.h"
+
+static bool enabled = false;
+static int16_t torque_command = 0;
+static int16_t speed_command = 0;
+Controls_Settings_T control_settings = {};
```

Comment:
Should be `volatile`. I doubt it would ever be optimized away, but it should be done.

Reply:
:+1:

Reply:
:+1: